Makefile

```diff
@@ -22,8 +22,8 @@ redeploy-container:
 	docker-compose down --remove-orphans
 	docker-compose up --force-recreate -d
 	$(call wait_for_container)
-	docker exec -e DISABLE_DATABASE_ENVIRONMENT_CHECK=1 mapknitter bash -lc \
-												"bundle exec rails db:drop && \
-												 bundle exec rails db:create && \
-												 bundle exec rails db:schema:load && \
-												 bundle exec rails db:migrate"
+	docker-compose run -e "DISABLE_DATABASE_ENVIRONMENT_CHECK=1" --rm web bash -lc \
```

Comment:
I think you should use `exec` here instead of `run`, because we want to run the following rails commands inside the same container previously created by the `docker-compose up --force-recreate -d`. When we use `run` it'll create another container.

Reply:
OK, thanks for the input! I'll wait for @icarito to reply here and for a resolution, before merging. Thanks!

Reply:
OK, I made the change, let's see how it does; i'll push it to unstable as well.